content/_data/authors/markewaite.adoc

```diff
@@ -0,0 +1,8 @@
+---
+name: "Mark Waite"
+twitter: MarkEWaite
+github: markewaite
+---
+
+One of the maintainers of the git plugin and long-time Jenkins user.
+---
```

Comment:
Interesting.  That last line is in the author section for Brian Dawson.  I'll remove it.